javascript/apps/taiga/src/app/modules/project/feature-kanban/models/edit-status.model.ts

```diff
@@ -9,6 +9,7 @@
 import { Status } from '@taiga/data';
 
 export interface EditStatus {
+  id: Status['id'];
   color: Status['color'];
   name: Status['name'];
   slug: Status['slug'];
```

Comment:
WorkflowStatus don't have `slug` attribute in the database anymore. Do you need this attr in the frontend?

Reply:
Deleted as we don't need it anymore